plugin-catalog/src/components/plugins/InstalledList.tsx

```diff
@@ -81,8 +152,25 @@ export function PluginInstalledList() {
       }
     }
 
+    function fetchOtherInstalledPlugins() {
+      const storedPlugins = localStorage.getItem('headlampPluginSettings');
```

Comment:
Is this the correct way to get the plugins? I think there's currently an issue related to this, since it mentions plugins settings but we have the package.json stored there, although it's likely a bug.
See #[this comment](https://github.com/headlamp-k8s/headlamp/pull/2578#issuecomment-2491732968).